proxy/src/grpc/sql_query.rs

```diff
@@ -140,6 +152,9 @@ impl<Q: QueryExecutor + 'static> Proxy<Q> {
                             error!("Failed to send record batches resp in stream sql query");
                             break;
                         }
+                        GRPC_HANDLER_COUNTER_VEC
```

Comment:
Maybe we should collect the sum of the number of rows of all the queried batches.

Reply:
this metric is in the forloop, so it can collect all rows, does it mean rows of forwarded queried?